addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -125,6 +125,21 @@
                 <span t-field="channel.product_id.list_price" t-options="{'widget': 'monetary', 'display_currency': product_info['currency_id']}"/>
             </div>
         </div>
+        <t t-if="channel.prerequisite_channel_ids and not channel.has_completed_prerequisites">
+            <small t-if="len(channel.prerequisite_channel_ids) == 1" class="text-center">
+                Prerequisite:
+                <a t-attf-href="/slides/{{slug(channel.prerequisite_channel_ids)}}">
+                    <span t-out="channel.prerequisite_channel_ids.name"/>
+                </a>
+            </small>
+            <small t-else="" class="text-center">
+                There are some
+                <a href="#" class="o_wslides_js_prerequisite_course"
+                   t-att-data-channels="json.dumps([{'id': c.id, 'name': c.name} for c in channel.prerequisite_channel_ids])">
```

Comment:
Can we have "course_id" and "course_name" for the keys? A bit easier to grep.

Also "course" instead of "c".

(I think you can multiple attributes to avoid getting a huge line).